apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -79,6 +80,16 @@ namespace MWRender
             }
             else
                 stateset->removeAttribute(osg::StateAttribute::POLYGONMODE);
+
+            osg::Uniform *borderUniform = stateset->getUniform("borders");
+
+            if (!borderUniform)
+            {
+                borderUniform = new osg::Uniform("borders",(int) 0);
+                stateset->addUniform(borderUniform);
```

Comment:
I tried adding the uniform only to the terrain node in ../components/terrain/world.cpp and then get it here with `getUniform` from the stateset, but I found this method is only called for the sceneRoot node, so I always got NULL from it. So now I'm simply setting this uniform for the whole sceneRoot node, but I'd like to know if there is a better way, i.e. only have the uniform on the terrain. Or is this OK?